x-pack/test/api_integration/apis/security/privileges.ts

```diff
@@ -62,7 +62,7 @@ export default function ({ getService }: FtrProviderContext) {
             logs: ['all', 'read'],
             observabilityCases: ['all', 'read'],
             uptime: ['all', 'read'],
-            apm: ['all', 'read'],
+            apm: ['all', 'read', 'minimal_all', 'minimal_read', 'alert_manage'],
```

Comment:
I expect this test will need updating with the transition from the single sub-feature privilege checkbox to the all/read split that we recently decided on